controller-server/src/main/java/com/yahoo/vespa/hosted/controller/Application.java

```diff
@@ -129,8 +130,8 @@ public Application withProjectId(long projectId) {
         return new Application(id, deploymentSpec, validationOverrides, deployments, deploymentJobs.withProjectId(projectId), deploying, outstandingChange);
     }
 
-    public Application withJiraIssueId(Optional<String> jiraIssueId) {
-        return new Application(id, deploymentSpec, validationOverrides, deployments, deploymentJobs.withJiraIssueId(jiraIssueId), deploying, outstandingChange);
+    public Application withIssueId(IssueId issueId) {
```

Comment:
Why not `with(IssueId issue)`? Existing code overloads when possible.

Reply:
Yes, good idea. It had to be named when it was not a custom class; not so anymore!